common/src/keyids.rs

```diff
@@ -14,17 +14,25 @@ Abstract:
 use caliptra_drivers::KeyId;
 
 #[cfg(feature = "rom")]
-pub const KEY_ID_UDS: KeyId = KeyId::KeyId0;
+pub const KEY_ID_ECDSA_UDS: KeyId = KeyId::KeyId0;
```

Comment:
As noted in https://github.com/chipsalliance/caliptra-sw/pull/1730, we do not need to allocate any extra key IDs for algorithm agility. We can condition the CDI differently when we turn it into a key, because we always pass it through a KDF first.

Reply:
With the new design, there is a single UDS for deriving the CDI.